app/models/classroom.rb

```diff
@@ -25,7 +26,7 @@ class Classroom < ApplicationRecord
   has_many :student_classrooms
   has_many :users, through: :student_classrooms
 
-  validates :name, presence: true
+  validates :subject, :group, presence: true
   validate :user_role
 
```

Comment:
please add method `name` that returns grade, group and subject all together
FE could then use this `name` readied method as opposed to creating it themselves, so we would need to pass it in the controller